stubs/caldav/caldav/objects.pyi

```diff
@@ -160,9 +161,9 @@ class CalendarObjectResource(DAVObject):
         self: Self, no_overwrite: bool = ..., no_create: bool = ..., obj_type: Any | None = ..., if_schedule_tag_match: bool = ...
     ) -> Self: ...
     data: Any
-    vobject_instance: _VBase
+    vobject_instance: VBase
     icalendar_instance: Any
-    instance: _VBase
+    instance: VBase
```

Comment:
I'm not familiar with vobject, but `VBase` sounds like a base class. Can we use a more specific type for this?

Reply:
Unfortunately not. Per https://github.com/python-caldav/caldav/blob/f30b574d099688760fe4ff6ee7ee13a5f82c5f08/caldav/objects.py#L1183-L1187:

```python
class CalendarObjectResource(DAVObject):
    """
    Ref RFC 4791, section 4.1, a "Calendar Object Resource" can be an
    event, a todo-item, a journal entry, or a free/busy entry
    """
```